lib/libzpool/uzfs_mgmt.c

```diff
@@ -334,6 +337,9 @@ uzfs_open_dataset_init(spa_t *spa, const char *ds_name, zvol_state_t **z)
 	zv->zv_volblocksize = block_size;
 	zv->zv_volsize = vol_size;
 
+	/* On boot, mark zvol status health */
+	zv->zv_status = ZVOL_STATUS_HEALTHY;
```

Comment:
on boot, we are supposed to mark zv as DEGRADED

Reply:
ok